pkg/controller/tuned/tuned_controller.go

```diff
@@ -278,15 +294,44 @@ func createCustomResource(mgr manager.Manager) error {
 	return nil
 }
 
+func addOwnerReference(meta *metav1.ObjectMeta, tuned *tunedv1alpha1.Tuned) []metav1.OwnerReference {
+	var bController bool = false
+	if tuned.Name == "default" {
+		bController = true
+	}
+
+	if meta.OwnerReferences == nil {
+		meta.OwnerReferences = []metav1.OwnerReference{}
+	} else {
+		for _, owner := range meta.OwnerReferences {
+			if owner.UID == tuned.UID {
```

Comment:
Is the UID safe enough to compare versus the `GroupVersion` comparison like in `referSameObject()`?

Reply:
Adding to TODO to look into.